QuestPDF/Drawing/FontManager.cs

```diff
@@ -100,5 +105,13 @@ internal static SKFontMetrics ToFontMetrics(this TextStyle style)
         {
             return FontMetrics.GetOrAdd(style.FontMetricsKey, key => style.ToPaint().FontMetrics);
         }
+
+        private static float GetTextScale(TextStyle style)
+        {
+            if (style.FontPosition == FontPosition.Superscript || style.FontPosition == FontPosition.Subscript)
```

Comment:
> Maybe we can hide this logic by moving it to the TextBlockSpan element? This way we can also avoid creating the ToNormalizedFontMetrics method. It may confuse us in the future.

I removed `ToNormalizedFontMetrics`. But i am not sure how to move the scaling of the text size to the `TextBlockSpan`. If we modify the text size in `TextBlockSpan` and pass the style to the canvas, will there be issues with the caching functionality?